This PR is not ready; with the current state, I get weird errors such as 
```
...
/Users/rasmus/Downloads/tensorflow/tensorflow/contrib/makefile/gen/protobuf-host/bin/protoc  tensorflow/core/util/memmapped_file_system.proto --cpp_out /Users/rasmus/Downloads/tensorflow/tensorflow/contrib/makefile/gen/host_obj/
tensorflow/tools/git/gen_git_source.sh tensorflow/core/util/version_info.cc
ifeq (x86_64,x86_64)
ifeq (x86_64,x86_64)
ifeq (x86_64,x86_64)
ifeq (x86_64,x86_64)
ifeq (x86_64,x86_64)
ifeq (x86_64,x86_64)
ifeq (x86_64,x86_64)
ifeq (x86_64,x86_64)
/bin/bash: -c: line 0: syntax error near unexpected token `x86_64,x86_64'
/bin/bash: -c: line 0: `ifeq (x86_64,x86_64)'
/bin/bash: -c: line 0: syntax error near unexpected token `x86_64,x86_64'
/bin/bash: -c: line 0: `ifeq (x86_64,x86_64)'
ifeq (x86_64,x86_64)
/bin/bash: -c: line 0: syntax error near unexpected token `x86_64,x86_64'
/bin/bash: -c: line 0: `ifeq (x86_64,x86_64)'
make: *** [tensorflow/contrib/makefile/Makefile:914: /Users/rasmus/Downloads/tensorflow/tensorflow/contrib/makefile/gen/host_obj/tensorflow/contrib/makefile/downloads/absl/absl/strings/substitute.o] Error 2
make: *** Waiting for unfinished jobs....
make: *** [tensorflow/contrib/makefile/Makefile:914: /Users/rasmus/Downloads/tensorflow/tensorflow/contrib/makefile/gen/host_obj/tensorflow/contrib/makefile/downloads/absl/absl/strings/str_split.o] Error 2
make: *** [tensorflow/contrib/makefile/Makefile:914: /Users/rasmus/Downloads/tensorflow/tensorflow/contrib/makefile/gen/host_obj/tensorflow/contrib/makefile/downloads/absl/absl/strings/charconv.o] Error 2
/bin/bash: -c: line 0: syntax error near unexpected token `x86_64,x86_64'
/bin/bash: -c: line 0: `ifeq (x86_64,x86_64)'
make: *** [tensorflow/contrib/makefile/Makefile:914: /Users/rasmus/Downloads/tensorflow/tensorflow/contrib/makefile/gen/host_obj/tensorflow/contrib/makefile/downloads/absl/absl/strings/numbers.o] Error 2
/bin/bash: -c: line 0: syntax error near unexpected token `x86_64,x86_64'
/bin/bash: -c: line 0: `ifeq (x86_64,x86_64)'
make: *** [tensorflow/contrib/makefile/Makefile:914: /Users/rasmus/Downloads/tensorflow/tensorflow/contrib/makefile/gen/host_obj/tensorflow/contrib/makefile/downloads/absl/absl/strings/string_view.o] Error 2
/bin/bash: -c: line 0: syntax error near unexpected token `x86_64,x86_64'
/bin/bash: -c: line 0: `ifeq (x86_64,x86_64)'
```

when running `build_all_linux.sh`, as if bash were somehow trying to parse the makefile. Anyone got an idea where this can come from?

--- a/tensorflow/contrib/makefile/Makefile
+++ b/tensorflow/contrib/makefile/Makefile
@@ -911,15 +911,15 @@ $(PROTO_TEXT_OBJS) : $(PROTO_TEXT_PB_H_FILES)
 # Ensures we link CoreFoundation as it is used for time library when building
 # for Mac and iOS
 ifeq ($(TARGET),OSX)
-	ifeq ($(HOST_ARCH),x86_64)
-		HOST_LDOPTS += -framework CoreFoundation
-		LIBS += -framework CoreFoundation
-	endif
+  ifeq ($(HOST_ARCH),x86_64)
+    HOST_LDOPTS += -framework CoreFoundation
+    LIBS += -framework CoreFoundation
+  endif
 endif
 ifeq ($(TARGET),IOS)
-	ifeq ($(IOS_ARCH),X86_64)
-		HOST_LDOPTS += -framework CoreFoundation
-	endif
+  ifeq ($(IOS_ARCH),X86_64)
+    HOST_LDOPTS += -framework CoreFoundation
+  endif
 endif
 
 # Runs proto_text to generate C++ source files from protos.

